internal/services/containers/kubernetes_cluster_resource_test.go

```diff
@@ -738,3 +738,42 @@ resource "azurerm_kubernetes_cluster" "test" {
 }
   `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, controlPlaneVersion)
 }
+
+func (KubernetesClusterResource) customCaTrustCertificates(data acceptance.TestData, fakeCertsList []string) string {
+	return fmt.Sprintf(`
+%s
+data "azurerm_kubernetes_cluster" "test" {
+  name                = azurerm_kubernetes_cluster.test.name
+  resource_group_name = azurerm_kubernetes_cluster.test.resource_group_name
+}
+`, KubernetesClusterResource{}.customCATrustCertificates(data, fakeCertsList))
+}
+
+func TestAccKubernetesCluster_customCaTrustCerts(t *testing.T) {
```

Comment:
In addition to moving this into `kubernetes_cluster_other_resource_test.go` where the test config lives, we group the tests and the test configs together, so could you move this up in the file to where the other tests are defined?